src/macaron/resources/policies/datalog/check-github-actions.dl.template

```diff
@@ -0,0 +1,8 @@
+#include "prelude.dl"
+
+Policy("github_actions_vulns", component_id, "GitHub Actions Vulnerability Detection") :-
+  check_passed(component_id, "mcn_githubactions_vulnerabilities_1").
```

Comment:
Please consistently use 4 spaces for indentation. This applies to all of the policy templates.